addOns/oast/src/main/java/org/zaproxy/addon/oast/ExtensionOast.java

```diff
@@ -71,8 +73,11 @@ public class ExtensionOast extends ExtensionAdaptor {
             Collections.unmodifiableList(Arrays.asList(ExtensionNetwork.class));
 
     private final Map<String, OastService> services = new HashMap<>();
-    private final ReferenceMap alertCache =
-            new ReferenceMap(AbstractReferenceMap.HARD, AbstractReferenceMap.SOFT);
+
+    @SuppressWarnings("unchecked")
+    private final Map<Object, Object> alertCache =
```

Comment:
btw, since the warning is being suppressed it could use the actual types and avoid the cast later.